pubspec.yaml

```
@@ -19,20 +19,20 @@ dependencies:
   gap: ^3.0.1
   hooks_riverpod: ^2.4.0
   # Ensure to keep  `/src/widgets/reactive_text_field/reactive_text_field.dart` in sync
-  reactive_forms: ^16.1.1
-  reactive_forms_annotations: ^4.0.0
+  reactive_forms: ^17.0.1
```

Please either do:
```suggestion
  reactive_forms: ^16.1.1
```

or update the `/src/widgets/reactive_text_field/reactive_text_field.dart` with changes in https://raw.githubusercontent.com/joanpablo/reactive_forms/refs/tags/17.0.1/lib/src/widgets/reactive_text_field.dart